modules/002-deckhouse/templates/deckhouse-leader-service.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: deckhouse-leader
+  namespace: d8-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "deckhouse")) | nindent 2 }}
+spec:
+  publishNotReadyAddresses: true
+  selector:
+    app: deckhouse
+    leader: true
```

Comment:
```suggestion
    leader: "true"
```

Reply:
Just to add some safety. `deckhouse` label could be interpreted only as string, while the `leader` label could be interpreted as boolean. You will get an error:
```
 Service in version "v1" cannot be handled as a Service: json: cannot unmarshal bool into Go struct field ServiceSpec.spec.selector of type string
 ```

Reply:
For example, you have to to like this with some values, like `controllerVersion`, which is string, but if you set `1.10` there, kubernetes will interpret it like a float

Reply:
At the same time, if we do `kubectl get pods -l leader=true`, kubectl will case this values to the string

Reply:
Thank you! 
Now it has become clearer.